templates/components/fat-footer/saa-fat-footer.html.twig

```diff
@@ -1,4 +1,4 @@
-{# Glue code for Decanter's component #}
+{# Glue code for Decanter's component  #}
```

Comment:
I'm not sure I can accept this.

Reply:
Haha, going to close this as it was a test. :)